File: extensions/api/client_example.java
import java.net.URI;
import java.net.http.HttpClient;
import java.net.http.HttpRequest;
import java.net.http.HttpResponse;

public class VentyClient {
    public static void main(String[] args) throws Exception {
        String json = "{\"command\": \"hello from Java\"}";
        
        HttpClient client = HttpClient.newHttpClient();
        HttpRequest request = HttpRequest.newBuilder()
                .uri(URI.create("http://localhost:8888/command"))
                .header("Content-Type", "application/json")
                .POST(HttpRequest.BodyPublishers.ofString(json))
                .build();

        HttpResponse<String> response = client.send(request, HttpResponse.BodyHandlers.ofString());
        System.out.println("Venty Response: " + response.body());
    }
}

File: extensions/api/client_example.js
async function sendCommand(cmd) {
    const response = await fetch('http://localhost:8888/command', {
        method: 'POST',
        headers: { 'Content-Type': 'application/json' },
        body: JSON.stringify({ command: cmd })
    });
    const data = await response.json();
    console.log("Venty Response:", data);
}

sendCommand("hello from JavaScript");
